data-prepper-plugin-framework/src/test/java/org/opensearch/dataprepper/plugins/test/TestComponent.java

```diff
@@ -0,0 +1,7 @@
+package org.opensearch.dataprepper.plugins.test;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Let's use `javax.inject` packages. We should encourage this as the primary annotation to use so that plugins are not coupled with Spring as the DI framework.

Reply:
Agree. It seems like the overall code is not standardized to use this one style. But at least the new code should start following this practice. Changed it to `javax.inject.Named` annotation.